### PR TITLE
Vault name instead of vault symbol inside descriptions

### LINKED ISSUE #118: Bug: The strategies in the description seem to add yv to the token
### What happened 🆘
Yearn watch is appending yv to the strategies deposit token when it should not. 

It should be looking for {{token}} from our ydeamon repo (https://github.com/yearn/ydaemon/blob/main/data/meta/strategies/1/ConvexReinvest.json#L4)

### To Reproduce 🔂
Steps to reproduce the behavior: go to yearn.watch or next.yearn.watch and look at any strategy even non-curve ones. 

### More 🔎
What it should look like (vaults.yearn.finance)
image
What I see
image

### PR DESCRIPTION
**Pull request by ai-slave**
fixes #118

### CODE REVIEW

**Comment by Majorfi** [comment]
Hey @ai-slave!
Thank you for your PR!
Could you fix the [type error](https://github.com/yearn/yearn-watch/actions/runs/3536556920/jobs/5940498371)?

**Comment by ai-slave** [comment]
Yeah, sorry push didn't went trough yesterday :(

**Comment by ai-slave** [comment]
Seems good now

**Review by Majorfi — APPROVED** [review]

File: components/sections/strategies/SectionAbout.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import {parseMarkdown}  from '@yearn-finance/web-lib/utils';
 
 type	TSectionAbout = {currentVault: TVault, currentStrategy: TStrategy | undefined};
 const	SectionAbout = memo(function SectionAbout({currentVault, currentStrategy}: TSectionAbout): ReactElement {
+	console.log(currentVault);
```
Comment (Majorfi) [inline components/sections/strategies/SectionAbout.tsx]:
```suggestion
```

**Review by Majorfi — CHANGES_REQUESTED** [review]

**Review by Majorfi — APPROVED** [review]

**Pull request merged**